include/sampling/random_point_generators.hpp

```diff
@@ -204,5 +204,78 @@ struct LogconcaveRandomPointGenerator
 };
 
 
+template
+<
+    typename Walk
+>
+struct ExponentialRandomPointGenerator
```

Comment:
Apply naming consistency if needed.

Reply:
I think this is consistent with the other structure names.